src/app/components/toolbar/toolbar.component.html

```diff
@@ -0,0 +1,4 @@
+<mat-toolbar>
+    <span>{{title}}</span>
+    <button mat-raised-button> Open discovery packs</button>
+    </mat-toolbar>
```

Comment:
This looks like a incorrectly indented tag

Reply:
You should also remove the open button as it is no longer being used

Reply:
won't it be used when I add the discovery drawer?

Reply:
Possibly, but that will be a hamburger menu button, not a material button with text

Reply:
ok thats done